src/screens/Account.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import PageTitle from "../components/PageTitle";
+import { EDIT_CANDIDATE_FIELDS } from "../mocks";
+import CandidateForm from "../components/CandidateForm";
+
+function Account() {
+  const dataChanged = (data) => {
+    console.log(data);
+  };
+  return (
+    <>
+      <PageTitle>
+        <h3>Edit account</h3>
+      </PageTitle>
+      <CandidateForm inputs={EDIT_CANDIDATE_FIELDS} action={dataChanged} />
```

Comment:
you can change the name of 'action' prop to 'onSubmit' or 'handleSubmit' to be more readable